Source/WebCore/page/FrameTree.cpp

```diff
@@ -491,6 +491,14 @@ AbstractFrame& FrameTree::top() const
     return *frame;
 }
 
+unsigned FrameTree::depth() const
+{
+    unsigned depth = 0;
+    for (auto* parent = &m_thisFrame; parent; parent = parent->tree().parent())
+        depth++;
```

Comment:
We could also write a function isDepthGreaterThanOrEqualTo that returns early, but since we are using this to limit the depth, I think they're basically equivalent in this case.